### PR TITLE
feat: add more docs about security

### PR DESCRIPTION
**Pull request by QuentinN42**
Greetings!

I work as a DevSecOps at Escape, where we operate a tool for testing the security of GraphQL. Securing web applications is crucial, and our objective is to simplify the process of adopting security best practices.

## Description

I have improve the existing documentation in the security section by adding DAST tools to test the API against basic security threats.

## Preview Link

image
image
image

## Types of changes

- [ ] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [x] Something else : **improve current documentation**.

## Checklist

- [x] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I have added a preview link.

### CODE REVIEW

**Comment by letzya** [comment]
@QuentinN42 Thank you for your contribution! We'll review it and update you!

File: tyk-docs/content/basic-config-and-security/security/dast.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+before it even reaches a production environment.
+
+#### Security testing tools for REST
+
+If you use REST APIs, you can use the following tools to test your APIs.
+To allow a robot to scan your app, you will need to retrieve the [openapi schema]({{< ref "getting-started/key-concepts/openapi-specification" >}}) of your API.
+
+##### openapi.security
+
+[openapi.security](https://openapi.security/) allow you to scan your OpenAPI specification for security issues.
```
Comment (letzya) [inline tyk-docs/content/basic-config-and-security/security/dast.md:23]:
```suggestion
[openapi.security](https://openapi.security/) allow you to scan your OpenAPI document for security issues.
```

File: tyk-docs/content/basic-config-and-security/security/dast.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+As GraphQL is a query language, it allows users to use a wider pannel of inputs than traditional REST APIs.
+
+Due to this feature, GraphQL APIs are inherently prone to various security risks, but they can be reduced by taking appropriate precautions.
+Neglecting them can expose the API to vulnerabilities like credential leakage or denial of service attacks.
+
+If you are using a graphql API, you may want to use one of the following tools to test your API.
+
+##### graphql.security
+
+[graphql.security](https://graphql.security/) is a free, quick graphql security testing tool, allowing you to quickly assess the most common vulnerabilities in your application.
```
Comment (letzya) [inline tyk-docs/content/basic-config-and-security/security/dast.md:36]: Couldn't see the Ts&Cs
image

**Comment by letzya** [comment]
@QuentinN42 Thanks again for this PR. ATM we don't have a page of recommended commercial tools. It could however fit into a nice blog on social media :)

File: tyk-docs/content/basic-config-and-security/security/dast.md (hunk 2 above)
Comment (QuentinN42) [reply tyk-docs/content/basic-config-and-security/security/dast.md:36]: https://graphql.security/disclaimer

**Pull request closed without merging**